mxnet-bot/LabelBotFullFunctionality/LabelBot.py

```diff
@@ -237,7 +237,9 @@ def predict_label(self, issue_num):
             message = "Hey, this is the MXNet Label Bot. \n Thank you for submitting the issue! I will try and " \
                       "suggest some labels so that the appropriate MXNet community members can help " \
                       "resolve it. \n "
-        message += 'Here are my recommended labels: {}'.format(', '.join(predicted_labels))
+        if predicted_labels:
```

Comment:
Can you change it to : 
``` if predicted_labels is not None and len(predicted_labels) > 0: ```

Reply:
That's equivalent. Harshs approach is right way how to check for a non empty list in python

Reply:
Aah. Seems like you are right. This is one those python magics.

https://stackoverflow.com/questions/53513/how-do-i-check-if-a-list-is-empty